CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ option(PEDANTIC_WARNINGS "Compile with pedantic warnings" OFF)
 option(BUILD_WITH_CODE_COVERAGE "Enable gcov code coverage" OFF)
 option(BUILD_GENIVI "Set to ON to compile with SWM and RVI gateway support" OFF)
 option(BUILD_OSTREE "Set to ON to compile with ostree and Uptane support" OFF)
+option(BUILD_SQLITE "Set to ON to compile with sqlite3" OFF)
```

Comment:
What do you think about making this a hard requirement? sqlite is pretty small, I suspect it is quite common on embedded systems already